src/server.c

```diff
@@ -2787,6 +2787,9 @@ void initListeners(void) {
         listener->bindaddr_count = 1;
         listener->ct = connectionByType(CONN_TYPE_UNIX);
         listener->priv = &server.unixsocketperm; /* Unix socket specified */
+        if(server.unixsocketgroup != NULL){
+            listener->group = server.unixsocketgroup; /* Unix socket group specified */
+        }
```

Comment:
```suggestion
        listener->group = server.unixsocketgroup;
```
If it's not set it's NULL, so it should be okay to always just set this. I prefer this so it's clearer what the value is.